src/templates/aids/_share_modal.html

```diff
@@ -0,0 +1,47 @@
+{% load i18n %}
+<div id="share-modal" class="modal">
+    <div class="modal-dialog">
+        <section>
+            <header>
+                <h1>{{ _('Share') }}</h1>
+            </header>
+
+            <div class="content">
+
+                <div class="share-target">
+                    <div class="col">
+                        <div class="fb-share-button" data-href="{{ request.build_absolute_uri }}" data-size="small">
+                            <a target="_blank" href="https://www.facebook.com/sharer/sharer.php?u={{ request.build_absolute_uri|urlencode}}&amp;src=sdkpreparse" class="fb-xfbml-parse-ignore">
+                                <img src="/static/img/logo_facebook.svg" alt="facebook">
+                            </a>
+                        </div>
+                    </div>
+                    <div class="col">
+                        <a target="_blank" href="https://www.linkedin.com/sharing/share-offsite/?url={{ request.build_absolute_uri }}">
+                            <img src="/static/img/logo_linkedin.svg" alt="linkedin">
+                        </a>
+                    </div>
+                    <div class="col">
+                        <a class="email" href="mailto:?subject=Votre%20recherche%20sur%20Aides%20territoires&body={{ request.build_absolute_uri }}">
+                            <img src="/static/img/logo_mail.svg" alt="mail">
+                        </a>
+                    </div>
+                    <div class="col">
+                        <a href="https://twitter.com/intent/tweet?status={{ request.build_absolute_uri|urlencode}}" target="_blank" onclick="window.open(this.href,'window','width=640,height=480,resizable,scrollbars') ;return false;">
```

Comment:
Ce lien ne marche pas chez moi. Sur Twitter, on a plus d'option pour pré-remplir le tweet. On peut utiliser les paramètres : 

 - text : pourrait être le nom de l'aide
 - url : l'url de la page courante
 - via : mentionner le compte Aides territoires (`@AidesTerr`)

Source : https://developer.twitter.com/en/docs/twitter-for-websites/tweet-button/overview